clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -507,6 +507,7 @@
     private final Metadata metadata;
     private final long retryBackoffMs;
     private final long requestTimeoutMs;
+    private final boolean autoCommitEnabled;
```

Comment:
Hmm... But the value is not actually read anywhere, right? Wouldn't the tests just check that `maybeAutoCommitOffsetsAsync` gets invoked (regardless of this field)?